### PR TITLE
Remove contact details section from embedded groups

### PR DESCRIPTION
**Pull request by KeirOsborn**
Part of task #1783

"remove the contact details"

This pull enacts the above. I.e. It removes the contact details from a group page when that page is being viewed in an embedded setting. 

What should we test?
Tests in spec/features/consumer/shopping/embedded_groups_spec.rb

Also functionality can be manually tested using the embedded-group-preview.html file. i.e. by creating a group and then navigating to: http://localhost:3000/embedded-group-preview.html?your-group-name.

Release notes
Second of four subtasks. Does not solve issue #1783 by itself.

### CODE REVIEW

**Comment by daniellemoorhead** [comment]
@sauloperez @enricostano @Matt-Yorkley @oeoeaio @mkllnk putting out the call for a code review, this has been sitting around since late december so I think it's time it got some love ❤️

File: spec/features/consumer/shopping/embedded_groups_spec.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+feature "Using embedded shopfront functionality", js: true do
+
+  Capybara.server_port = 9999
```
Comment (sauloperez) [inline spec/features/consumer/shopping/embedded_groups_spec.rb]: It's not the firsr time I see this. Aren't we sure it's not already defined somewhere in the spec_helper.rb?

File: spec/features/consumer/shopping/embedded_groups_spec.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+  describe 'embedded groups' do
+	let(:enterprise) { create(:distributor_enterprise) }
+  	let!(:group) { create(:enterprise_group, enterprises: [enterprise], permalink: 'group1', on_front_page: true) }
+
+  	before do
+  	  Spree::Config[:enable_embedded_shopfronts] = true
+      Spree::Config[:embedded_shopfronts_whitelist] = 'localhost'
+
+      page.driver.browser.js_errors = false
```
Comment (sauloperez) [inline spec/features/consumer/shopping/embedded_groups_spec.rb]: Weird identation

File: spec/features/consumer/shopping/embedded_groups_spec.rb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+	end
+
+	it "doesn't display contact details when embedded" do
+	  expect(page).to have_selector 'iframe#group_test_iframe'
+
+	  within_frame 'group_test_iframe' do
+	    within 'div#group-page' do
+          expect(page).to have_no_selector 'div.contact-container'
+          expect(page).to have_no_content '#{group.address.address1}'
+	  	end
```
Comment (sauloperez) [inline spec/features/consumer/shopping/embedded_groups_spec.rb]: Weird identation here too

**Comment by sauloperez** [comment]
Thanks @daniellemoorhead!

**Pull request closed without merging**